src/tests/Common/Directory.Build.targets

```
@@ -187,7 +187,7 @@
           TargetDir="wasm-test-runner/"/>
 
         <RunTimeDependencyCopyLocal
-          Include="$(MonoProjectRoot)\wasm\test-main.js"
+          Include="$(MonoProjectRoot)\browser\test-main.js"
```

Can we use `BrowserProjectRoot`?

```suggestion
          Include="$(BrowserProjectRoot)\test-main.js"
```